Sources/SRGMediaPlayer/SRGTimeSlider~ios.m

```diff
@@ -23,6 +24,8 @@
 static NSString *SRGTimeSliderFormatter(NSTimeInterval seconds)
 {
     NSCAssert(seconds >= 0, @"A non-negative number of seconds is expected");
+    NSCAssert(!isnan(seconds), @"A number of seconds is expected");
+    NSCAssert(!isinf(seconds), @"A non-infinite number of seconds is expected");
```

Comment:
Since you are now addressing these exceptional conditions directly I would suggest we remove assertions entirely. Assertions should make it possible to identify obvious programming errors but here we obviously needed to be more resilient to cases that can actually happen in practice.

Reply:
Good point. Did it.